tests/BootstrapperTest.php

```diff
@@ -85,7 +85,7 @@
 test('cache data is separated', function () {
     config([
         'tenancy.bootstrappers' => [
-            CacheTenancyBootstrapper::class,
+            CacheTaggingBootstrapper::class,
```

Comment:
In an earlier version of this PR we used the prefix bootstrapper here, now it's changed "back" to the tagging bootstrapper. Maybe it'd be reasonable to test this with both bootstrappers (perhaps using a data provider with the bootstrapper class?)

Reply:
Same in `GlobalCacheTest` maybe

Reply:
> Maybe it'd be reasonable to test this with both bootstrappers

Agree